metron-analytics/metron-profiler/pom.xml

```diff
@@ -339,6 +339,7 @@
                                 </excludes>
                             </artifactSet>
                             <transformers>
+                                <transformer implementation="org.atteo.classindex.ClassIndexTransformer"/>
```

Comment:
That's only required for shaded jar so that the shading merges the indices.  I need to add some docs around "adding new stellar functions" for sure.